### ISSUE TITLE
Pillow Antialias depreciation warning

### ISSUE THREAD

**Issue by raidensakura** [DEViantUA/EnkaNetworkCard#2]
A warning is thrown by Pillow when trying to generate character card.
```
[15:16:29] WARNING  [py.warnings] C:\Users\Raiden\redenv\lib\site-packages\aioenkanetworkcard\src\utils\FunctionsPill.py:31: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
  new_image = imgs.resize((fixed_width, height), Image.ANTIALIAS)
```
Output of `python --version`
```                                                                                                                                          
Python 3.9.13
```
Result of `pip show pillow`:
```
Name: Pillow
Version: 9.4.0
Summary: Python Imaging Library (Fork)
Home-page: https://python-pillow.org
Author: Alex Clark (PIL Fork Author)
Author-email: aclark@python-pillow.org
License: HPND
Location: c:\users\raiden\redenv\lib\site-packages
Requires: 
Required-by: aioenkanetworkcard
```
Usage:
```py
        async def get_character_card(uid, char_name):
            async with encbanner.ENC(
                lang="en", splashArt=True, characterName=char_name
            ) as encard:
                ENCpy = await encard.enc(uids=uid)
                return await encard.creat(ENCpy, 2)
                
        async def generate_char_info(uid, char_name):
            """Generate character info"""
            with io.BytesIO() as image_binary, ctx.typing():
                char_card = await get_character_card(uid, char_name)
                if not char_card:
                    return await ctx.send("Character card not found.")
                temp_filename = str(time.time()).split(".")[0] + ".png"
                log.debug(char_card)
                char_card[uid][char_name].save(
                    image_binary, "PNG", optimize=True, quality=95
                )
                image_binary.seek(0)
                return await ctx.send(
                    file=discord.File(fp=image_binary, filename=temp_filename)
                )
```

Additionally, the Discord invite link on readme is also invalid.

**Comment by straw-hatking** [comment]
could u give the code

**Comment by straw-hatking** [comment]
and tell me your python version

**Comment by raidensakura** [comment]
> and tell me your python version

> could u give the code

Updated description, additionally created a PR to fix #3

**Comment by straw-hatking** [comment]
um the code has a problem in a line 
new_image = imgs.resize((fixed_width, height), Image.ANTIALIAS)
And could u like even add the part of code where the line is there

**Comment by DEViantUA** [comment]
Does the error interfere with the generation of the image or simply give out information in the logs that there will be a replacement on 2023.07?

> A warning is thrown by Pillow when trying to generate character card.
> 
> ```
> [15:16:29] WARNING  [py.warnings] C:\Users\Raiden\redenv\lib\site-packages\aioenkanetworkcard\src\utils\FunctionsPill.py:31: DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use LANCZOS or Resampling.LANCZOS instead.
>   new_image = imgs.resize((fixed_width, height), Image.ANTIALIAS)
> ```
> 
> Output of `python --version`
> 
> ```
> Python 3.9.13
> ```
> 
> Result of `pip show pillow`:
> 
> ```
> Name: Pillow
> Version: 9.4.0
> Summary: Python Imaging Library (Fork)
> Home-page: https://python-pillow.org
> Author: Alex Clark (PIL Fork Author)
> Author-email: aclark@python-pillow.org
> License: HPND
> Location: c:\users\raiden\redenv\lib\site-packages
> Requires: 
> Required-by: aioenkanetworkcard
> ```
> 
> Usage:
> 
> ```python
>         async def get_character_card(uid, char_name):
>             async with encbanner.ENC(
>                 lang="en", splashArt=True, characterName=char_name
>             ) as encard:
>                 ENCpy = await encard.enc(uids=uid)
>                 return await encard.creat(ENCpy, 2)
>                 
>         async def generate_char_info(uid, char_name):
>             """Generate character info"""
>             with io.BytesIO() as image_binary, ctx.typing():
>                 char_card = await get_character_card(uid, char_name)
>                 if not char_card:
>                     return await ctx.send("Character card not found.")
>                 temp_filename = str(time.time()).split(".")[0] + ".png"
>                 log.debug(char_card)
>                 char_card[uid][char_name].save(
>                     image_binary, "PNG", optimize=True, quality=95
>                 )
>                 image_binary.seek(0)
>                 return await ctx.send(
>                     file=discord.File(fp=image_binary, filename=temp_filename)
>                 )
> ```
> 
> Additionally, the Discord invite link on readme is also invalid.

**Comment by raidensakura** [comment]
> Does the error interfere with the generation of the image or simply give out information in the logs that there will be a replacement on 2023.07?

It doesn't since it's just a warning, but it gets annoying and better to change it early since it will be removed in 6 months anyway.

**Comment by DEViantUA** [comment]
Fixed in version 2.0.8
